examples/js/loaders/SVGLoader.js

```diff
@@ -630,7 +630,18 @@ THREE.SVGLoader.prototype = {
 
 			for ( var i = 0; i < array.length; i ++ ) {
 
-				array[ i ] = parseFloat( array[ i ] );
+				var number = array[ i ];
+
+				// Handle values like 48.6037.7
+				// TODO Find a regex for this
+
+				if ( number.indexOf( '.' ) !== number.lastIndexOf( '.' ) ) {
```

Comment:
😮

Reply:
Haha! Yeah, not super intuitive... But that's the simplest code I could come up with without creating a new function.